### PR TITLE
[JENKINS-34235] Wrong plugin short-name in the URL pattern

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-34235](https://issues.jenkins-ci.org/browse/JENKINS-34235)

@reviewbybees especially @amuniz

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public String getDescription() {
         }
 
         public String getIconFilePathPattern() {
-            return "plugin/workflow-images/:size/pipelinemultibranchproject.png";
+            return "plugin/workflow-multibranch/:size/pipelinemultibranchproject.png";
```
Comment (amuniz) [inline src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: Why was it `workflow-images` before?? Did that even exist at some point?

**Comment by amuniz** [comment]
:bee:

File: src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java (hunk 1 above)
Comment (recena) [inline src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: It was my wrong, maybe a copy&paste??? :panda_face:

Comment (recena) [inline src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: In the original PR was fine https://github.com/jenkinsci/pipeline-plugin/pull/376/files#diff-fc4cf5ce663b563407ec5283d4866c34R82

I found the problem. I had to do a replace of `multibranch` to adapt the patch file :sob:

**Comment by jtnord** [comment]
:bee:

**Pull request merged**